src/app/restaurantlist/restaurantlist.component.html

```diff
@@ -0,0 +1,19 @@
+<section *ngFor="let restaurant of restaurantList">
+    <div class="restaurant">
+      <div>
+        <p>Name: {{ restaurant.name }}</p>
+      </div>
+      <div>
+        <p>Description: {{ restaurant.description }}</p>
+      </div>
+      <div>
+        <p>Date of Creation: {{ restaurant.dateOfCreation }}</p>
+      </div>
+      <div>
+        <p>ID: {{ restaurant.id }}</p>
```

Comment:
Id is usually used only to have an identifier that is unique, meaning something that can let us differentiate between restaurants. It is never shown on the UI, so we can remove this.